lib/utils/encode_passcode.dart

```diff
@@ -1,9 +1,13 @@
 import 'dart:convert' show base64Encode, utf8;
 
-import 'package:crypto/crypto.dart' show sha256;
+import 'package:crypto/crypto.dart' show md5, sha256;
 
 String encodePasscode(String passcode) {
   final bytes = utf8.encode(passcode);
   final passcodeHash = sha256.convert(bytes);
   return base64Encode(passcodeHash.bytes);
 }
+
+String hashMD5(String input) {
```

Comment:
Consider renaming this file, now that it no longer only encodes the password

Reply:
Since you ended up not making use of gravatars, consider simply deleting the unused function as well